drivers/hmis/lib/form_data/tarrant_county/assessments/caseworker_exit.json

```diff
@@ -0,0 +1,611 @@
+{
+  "item": [
+    {
+      "text": "Client Data",
+      "type": "GROUP",
+      "link_id": "caseworker_exit_client_data",
+      "item": [
+        {
+          "text": "Assessment Date",
+          "type": "DATE",
+          "link_id": "caseworker_assessment_date",
+          "required": true,
+          "mapping": {
+            "field_name": "assessmentDate"
+          },
+          "assessment_date": true
+        },
+        {
+          "text": "Exit Date",
+          "type": "DATE",
+          "link_id": "caseworker_exit_date",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_date"
+          }
+        },
```

Comment:
This is a little weird, I'd think we could just show the `Enrollment.exitDate` in the interface as read-only, or remove this if its visible elsewhere. Let's clarify.